static/js/tools/map/compute_hook.ts

```diff
@@ -17,23 +17,14 @@
 // This module contains custom React hooks that makes computation for map chart.
```

Comment:
nit: compute_hooks.ts for the file name?

Reply:
Done